airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -253,3 +253,10 @@ def _get_secret(self, path_prefix, secret_id: str) -> Optional[str]:
                 secret_id,
             )
             return None
+        except self.client.exceptions.AccessDeniedException as e:
+            self.log.debug(
+                "An error occurred (AccessDeniedException) when calling the "
+                "get_secret_value operation: %s",
+                str(e),
+            )
```

Comment:
```suggestion
        except self.client.exceptions.AccessDeniedException:
            self.log.debug(
                "An error occurred (AccessDeniedException) when calling the "
                "get_secret_value operation",
                exc_info=True,
            )
```

`logging` has a built-in way to log an exception! 🙂

Reply:
@john-jac not sure  if you saw this comment?  would you mind either implementing the suggestion or stating why not?  i suspect   it was auto-dismissed after a rebase or something.

thanks

Reply:
oh looks like you resolved it i am not sure how i got on old commit never mind